Zoo/src/main/java/Zoo/Menu.java

```diff
@@ -0,0 +1,126 @@
+package Zoo;
+
+import Zoo.Birds.Crane;
+import Zoo.Birds.Crow;
+import Zoo.Birds.Peacock;
+import Zoo.Fishes.Hammerfish;
+import Zoo.Fishes.Sawfish;
+import Zoo.Fishes.Shark;
+import Zoo.Mammals.Elephant;
+import Zoo.Mammals.Lion;
+import Zoo.Mammals.Racoon;
+
+import java.util.Scanner;
+
+public class Menu {
+    public static void userMenu(){
+        Lion bonifaci = new Lion("Bonifaci");
+        Elephant dumbo = new Elephant("Dumbo");
+        Racoon rocky = new Racoon("Rocky");
+        Hammerfish tor = new Hammerfish("Tor");
+        Sawfish saw = new Sawfish("Saw");
+        Shark kilobite = new Shark("Kilobite");
+        Crane confucious = new Crane("Confucious");
+        Crow jet = new Crow("Jet");
+        Peacock handsome = new Peacock("Handsome");
+        int choice;
+        System.out.println("What would you like to see? Choose any options to get more information about animal:");
+        do {
+            Animal.getListOfAnimal();
+            System.out.println("0. Exit");
+            Scanner scanner = new Scanner(System.in);
+            choice =scanner.nextInt();
+            scanner.nextLine();
+
+            // Block of choice
+            switch (choice){
```

Comment:
Would be much shorter if we had a list of animals and just use the indexes to call different animals. This would make the code more extensible (add new animal to the list and nothing else to add)

Reply:
Implement it.